webui/templates/404.html

```diff
@@ -25,4 +25,6 @@ <h1 class="text-center display-2">404</h1>
 </div>
 <!-- container -->
 
+{% include "sponsors.html" %}
```

Comment:
Are you adding sponsors to all the pages? or some subset of pages? maybe this should be put in the base template.